### DISCUSSION TITLE
Why does the free version of Set-OutlookSignatures add a tagline to signatures? (was: "tagline in signatures makes OSS Version useless....")

### DISCUSSION THREAD

**Discussion by BiasF** [Set-OutlookSignatures/Set-OutlookSignatures#126] [Q&A] [answered]
### Issue happens in the latest release

- [X] I confirm that the issue happens in the latest release of Set-OutlookSignatures

### Previously solved issues and documentation

- [X] I have searched through previous issues and documentation, but have not found an answer to my issue

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

This function makes the "OpenSource" Version useless.

**Comment by GruberMarkus** [comment, marked as answer]
Hi @BiasF,

thanks for bringing this up.

The tagline should not be a surprise as it is mentioned popularly in the CHANGELOG and explained in the README file.

I assume you are looking for background information and I am happy to provide it. Let me know what you think about it and do not hesitate to ask should you have more questions.

## Why the tagline?
I initially created Set-OutlookSignatures to give back to the community by showing how to correctly script stuff that I have seen being done in wrong and incomplete ways over and over again:
- Efficient queries for nested Active Directory group membership,
- working with SID history,
- working with AD queries in the most complex environments and across trusts,
- parallel code execution in PowerShell,
- working with Graph,
- and - of course - a fresh approach on how to manage and deploy signatures for Outlook.

Since the free version of Set-OutlookSignatures has first been published in 2021, dozens of features have been added - quickly scroll through the CHANGELOG to get an idea of what I am talking about.<br>I invested more than a thousand hours of my spare time developing them, and I spent a whole lot of money setting up and maintaining different test environments. And I plan to continue doing so and keeping the core of Set-OutlookSignatures free and open source software.

You are probably an Exchange or client administrator, and as such you are part of the community I want to give something back to.

I do not expect or request thank yous from fellow admins, as our community lives from both giving and taking.

I draw the line where companies, rather than individuals, benefit one-sidedly. The tagline reminds companies that they benefit from open source software and that there is a way to ensure that Set-OutlookSignatures remains open source and is developed further by supporting it financially and at the same time gaining access to even more useful features.

By the way: Companies often make wrong assumptions about free and open source software. Open source software absolutely can contain closed source code. The term "open source" does not automatically imply free usage or even free access to the code. The permission to use software for free does not imply free support.

## Not sure if Set-OutlookSignatures is the right solution for your company?
The core of Set-OutlookSignatures is available free of charge as open-source software and can be used for as long and for as many mailboxes as your company wants.<br>All documentation is publicly available, and you can get free community support at GitHub or get first-class commercial support, training, workshops and more from [ExplicIT Consulting](https://explicitconsulting.at/open-source/set-outlooksignatures/).

For a small annual fee per mailbox, the [Benefactor Circle add-on](https://set-outlooksignatures.com/benefactorcircle) offers a whole bunch of additional features.<br>All documentation is publicly available, and the free 14-day trial version allows companies to test all additional features at no cost.

Your company is not sure whether the add-on will pay off?<br>Visit https://set-outlooksignatures.com/benefactorcircle#financial-benefits and learn how you can do the calculation tailored to the needs of your company.<br>Should your company come to the conclusion that the add-on does not pay off, it can still use the free and open source version of Set-OutlookSignatures.

**Comment by GruberMarkus** [comment]
Dear companies, please do not forget:
- Invest in the free and open-source software you depend on. Contributors are working behind the scenes to make open-source better for everyone. Give them the help and recognition they deserve.
- Sponsor the free and open-source software your teams use to keep your business running. Fund the projects that make up your software supply chain to improve its performance, reliability, and stability.

Being free and open-source software, Set-OutlookSignatures saves your company a remarkable amount of money compared to commercial software.

**Comment by GruberMarkus** [comment]
@BiasF: I am interested in your feedback, as you started the discussion about this topic.

**Comment by BiasF** [comment]
Hi,

you will get my answer later until I am back home.

Tobias

**Comment by GruberMarkus** [comment]
For anyone coming across this discussion: Set-OutlookSignatures has its own website now, and there is an FAQ related to the topic discussed here, giving a bigger picture and referring to additional information: '[Why the tagline?](https://set-outlooksignatures.com/faq/#why-the-tagline)'
